engine/execution/state/delta/view_test.go

```diff
@@ -225,7 +225,8 @@ func TestView_MergeView(t *testing.T) {
 		err = chView.Set(registerID2, "", "", flow.RegisterValue("carrot"))
 		assert.NoError(t, err)
 
-		v.MergeView(chView)
+		err = v.MergeView(chView)
+		assert.NoError(t, err)
```

Comment:
shouldn't all these asserts be `require`?

Reply:
they are both the same, isn't it ?